app/views/provider_interface/reports/index.html.erb

```diff
@@ -40,6 +40,9 @@
           <li>
             <%= govuk_link_to t('page_titles.provider.withdrawal_report'), provider_interface_reports_provider_withdrawal_report_path(provider_id: provider) %>
           </li>
+          <li>
+            <%= govuk_link_to t('page_titles.provider.reference_status_report'), provider_interface_reports_provider_reference_status_report_path(provider_id: @providers.first) %>
```

Comment:
Since this is iterating through a number of providers this should be `provider_id: provider`.

Then we'll need another link within the `@providers.one?` statement above. That is where we can pass the id in as `@providers.first`